source/install/requirements.rst

```diff
@@ -83,7 +83,7 @@ Server Software
 Mattermost Server Operating System
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Ubuntu 16.04, Ubuntu 18.04, Debian Buster, CentOS 6+, CentOS 7+, RedHat Enterprise Linux 6+, RedHat Enterprise Linux 7+, Oracle Linux 6+, Oracle Linux 7+
```

Comment:
We may not support versions after Ubuntu 18.04, or CentOS 8+, RedHat 8+, Oracle 8+. We should check that.

Reply:
How do I check that?

Reply:
I would try with the support team, if they're not sure, then SET.

(My guess is we don't support since they're not listed)

Reply:
From support: "We don't have official guides for them yet, but they should work similar if not the same unless the maintainers of said distros did some major rework between versions. Last time we looked at it it was the mysql version that was blocking updating the documentation"